<ui version="4.0" >
 <class>QAudioMixerTestBase</class>
 <widget class="QWidget" name="QAudioMixerTestBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QAudioMixerTest</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="0" >
      <widget class="QRadioButton" name="muteButton" >
       <property name="text" >
        <string>Mute</string>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QComboBox" name="comboBox" />
     </item>
     <item row="2" column="0" colspan="3" >
      <widget class="QSlider" name="horizontalSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3" >
      <widget class="QLabel" name="Title" >
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text" >
        <string>Audio Mixer Test</string>
       </property>
       <property name="scaledContents" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QPushButton" name="prevButton" >
       <property name="text" >
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>131</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="2" >
      <widget class="QPushButton" name="nextButton" >
       <property name="text" >
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3" >
      <widget class="QLabel" name="label" >
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>1</number>
       </property>
       <property name="midLineWidth" >
        <number>0</number>
       </property>
       <property name="text" >
        <string>Name of Current Element</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
